gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/RequestMapper.java

```diff
@@ -48,9 +83,7 @@ public static BrokerUserTaskCompletionRequest toUserTaskCompletionRequest(
             .withSubject(JwtAuthorizationBuilder.DEFAULT_SUBJECT)
             .withClaim(Authorization.AUTHORIZED_TENANTS, authorizedTenants)
             .encode();
-    brokerRequest.setAuthorization(authorizationToken);
-
-    return brokerRequest;
+    return authorizationToken;
```

Comment:
🙃 You can directly return the authorization above, no need to create a variable.